src/Diagnostics.RuntimeHost/Models/PostBodyModel.cs

```diff
@@ -12,6 +12,7 @@ public class DiagnosticSiteData
         public string NamespaceDescriptor { get; set; }
         public bool IsDefaultContainer { get; set; }
         public bool? IsLinux { get; set; }
+        public bool IsXenon => !string.IsNullOrWhiteSpace(Kind) && Kind.ToLower().Contains("xenon");
```

Comment:
This logic would need a change I believe after Joaquin's email?

Reply:
I pushed the latest update this morning